pkg/certs/cert-inspection/certgraphanalysis/collector.go

```diff
@@ -41,6 +48,7 @@ func isPlatformNamespace(nsName string) bool {
 }
 
 type configMapFilterFunc func(configMap *corev1.ConfigMap) bool
+type configMapRewriteFunc func(configMap *corev1.ConfigMap, nodes map[string]int) (string, string)
```

Comment:
Accepting nodes here is a losing game.  We'll be adding more and more parameters. Instead when you create a rewriter do something like


```
RewriteNodeNames(nodes []*corev1.Node) *certGenerationOptions{
  return &certGenerationOptions{
     // body here using those nodes across all secret names
  }
}
```